### ISSUE TITLE
Is it possible to use an email and password for this?

### ISSUE THREAD

**Issue by MJB1113** [RumbleMike/ValorantStreamOverlay#180]

**Comment by SoldierCantKill** [comment]
Nope, if there is it is strictly against valorant TOS because valorant login is Userpass to prevent people who spam API requests to try to find posible logins; less likely to get accounts if userpass.
